crates/bevy_input/src/gamepad.rs

```diff
@@ -307,12 +307,13 @@ pub enum ButtonSettingsError {
     },
 }
 
-/// Stores a connected gamepad's state and any metadata such as the device name.
+/// The [`Gamepad`] [`component`](Component) stores a connected gamepad's metadata such as the `name` and its [`GamepadButton`] and [`GamepadAxis`].
```

Comment:
It's not very conventional to put a doc link into self. Maybe write it like this?
```suggestion
/// Stores a connected gamepad's metadata such as the name and its [`GamepadButton`] and [`GamepadAxis`].
```